master/reference/advanced/etcd-rbac/users-and-roles.md

```diff
@@ -2,11 +2,9 @@
 title: Creating Users and Roles in etcd
 ---
 
-## Overview
-
-RBAC is *Role* Based Access Control so roles must be setup and also users
-that are given those roles must be created.  This document helps with the
-creation of both of those.
+In etcd, roles must be setup with access to the appropriate paths and also
```

Comment:
It might just be me, but  still find this a wording a little odd.  I'd probably have worded something like the text below, but what you have makes sense to me, so feel free to ignore.  

Only real comment is given that this is true for etcdv3 should we be talking about keys rather than paths.  Not sure WDYT?

```
Providing role based access control within etcd requires the following:
-  Creation of etcd roles which provide appropriate access to the specific set of etcd keys required by the role
-  Creation of etcd users who are assigned roles.

This guide provides useful information to help configure the etcd roles and users.
```